.github/workflows/e2e-tests.yaml

```diff
@@ -33,13 +39,8 @@ jobs:
         export ITER8=$(pwd)
         $ITER8/samples/setup/iter8-setup.sh
 
-    - name: Install Go
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.16
-    
     - name: Get iter8ctl
-      run: sudo GO111MODULE=on GOBIN=/usr/local/bin go get github.com/iter8-tools/iter8ctl@v0.1.6
+      run: go install github.com/iter8-tools/etc3/iter8ctl@latest
```

Comment:
I believe we should drop @latest (we want the latest *semantically versioned* release).

`go install github.com/iter8-tools/etc3/iter8ctl` is probably the right thing to do to achieve the above.

Reply:
`@latest` seems to be required: 

```bash
$ go install github.com/iter8-tools/etc3/iter8ctl
go install: version is required when current directory is not in a module
	Try 'go install github.com/iter8-tools/etc3/iter8ctl@latest' to install the latest version
```

Perhaps by moving etc3 into a subdirectory (possible after v0.8 integration) and creating modules in each we can solve this. But for now, no.

Reply:
I am able to install without `@latest`. I use the full `GO111MODULE=on ... ` command locally.

Reply:
This seemed to work when I tried locally. But I get an error when running the workflow. See https://github.com/iter8-tools/iter8/pull/987/checks?check_run_id=3666987855

Reply:
I am getting a different error ... https://github.com/iter8-tools/iter8/runs/3666504483?check_suite_focus=true

I thought this error was fixed ... anyway, it does seem to be getting the right install version for me though!

Reply:
I fixed this by moving the install of go earlier in the workflow.

Reply:
@kalantar https://pkg.go.dev/cmd/go -- use of `@latest` seems to be the right way to do this (latest does not refer to master branch of the Git repo; it refers to the latest semantically ordered version, which is the interpretation that makes sense and works for us).

So all good with the use of `@latest`. This is what we need. Sorry about creating a confusion in this regard.